.gitmodules

```diff
@@ -1,16 +1,25 @@
-[submodule "packages/protocol/lib/solmate"]
-	path = packages/protocol/lib/solmate
-	url = https://github.com/rari-capital/solmate
-	branch = v1.5.0
+# git submodule update --recursive --remote
+[submodule "packages/protocol/lib/forge-std"]
+	path = packages/protocol/lib/forge-std
+	url = https://github.com/foundry-rs/forge-std
+	tag = v1.7.6
 [submodule "packages/protocol/lib/openzeppelin-contracts-upgradeable"]
 	path = packages/protocol/lib/openzeppelin-contracts-upgradeable
 	url = https://github.com/OpenZeppelin/openzeppelin-contracts-upgradeable
-	branch = v4.8.2
+	tag = v4.8.2
 [submodule "packages/protocol/lib/openzeppelin-contracts"]
 	path = packages/protocol/lib/openzeppelin-contracts
 	url = https://github.com/OpenZeppelin/openzeppelin-contracts
-	branch = v4.8.2
-[submodule "packages/protocol/lib/forge-std"]
-	path = packages/protocol/lib/forge-std
-	url = https://github.com/foundry-rs/forge-std
-	branch = chore/v1.5.1
+	tag = v4.8.2
+[submodule "packages/protocol/lib/solady"]
+	path = packages/protocol/lib/solady
+	url = https://github.com/Vectorized/solady
+	tag = v0.0.166
+[submodule "packages/protocol/lib/p256-verifier"]
+	path = packages/protocol/lib/p256-verifier
+	url = https://github.com/taikoxyz/p256-verifier
+	commit = c8210cd99f9f81a58e25a6a8afdb992fefbd9748
+[submodule "packages/protocol/lib/automata-dcap-v3-attestation"]
+	path = packages/protocol/lib/automata-dcap-v3-attestation
+	url = https://github.com/automata-network/automata-dcap-v3-attestation
+	commit = 80eba1c014c193d4e83e07ac7d015cea8ebf1232
```

Comment:
@smtmfft is this the one we are using on our branch , right ?

Reply:
this is the latest commit, they don't have any release.

Reply:
All submodules should specify something (commit, tag, branch) so the code we use is deterministic.

Reply:
I know, but we need to compare this is the one we indeed using, because i brought over this from @smtmfft repo.

Reply:
Then we need to clone @smtmfft's repo, then use that github repo's URL as the submodule URL.

Reply:
Feel free to change the config in the .gitmodules file.